app/controllers/parties_controller.rb

```diff
@@ -4,6 +4,24 @@ def index
     render json: parties
   end
 
+  # show
+  def show
+    party = Party.find(params[:id])
+    render json: party.as_json(include: :user)
+  end
+
```

Comment:
What is going on here?

Reply:
We are setting up to have access to user: first name, last name from parties inside the controller.  This will allow us to show us the host's name